recipe/linux/activate.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Set `CUDA_HOME` in an activation script.
+
+# Backup environment variables (only if the variables are set)
+if [[ ! -z "${CUDA_HOME+x}" ]]
+then
+  export CUDA_HOME_CONDA_NVCC_BACKUP="${CUDA_HOME:-}"
+fi
+
+if [[ ! -z "${CUDA_PATH+x}" ]]
+then
+  export CUDA_PATH_CONDA_NVCC_BACKUP="${CUDA_PATH:-}"
+fi
+
+if [[ ! -z "${CFLAGS+x}" ]]
+then
+  export CFLAGS_CONDA_NVCC_BACKUP="${CFLAGS:-}"
+fi
+
+if [[ ! -z "${CPPFLAGS+x}" ]]
+then
+  export CPPFLAGS_CONDA_NVCC_BACKUP="${CPPFLAGS:-}"
+fi
+
+if [[ ! -z "${CXXFLAGS+x}" ]]
+then
+  export CXXFLAGS_CONDA_NVCC_BACKUP="${CXXFLAGS:-}"
+fi
+
+if [[ ! -z "${CMAKE_ARGS+x}" ]]
+then
+  export CMAKE_ARGS_CONDA_NVCC_BACKUP="${CMAKE_ARGS:-}"
+fi
+
+# Default to using $(cuda-gdb) to specify $(CUDA_HOME).
+if [[ -z "${CUDA_HOME+x}" ]]
+then
+    CUDA_GDB_EXECUTABLE=$(which cuda-gdb || exit 0)
+    if [[ -n "$CUDA_GDB_EXECUTABLE" ]]
+    then
+        CUDA_HOME=$(dirname $(dirname $CUDA_GDB_EXECUTABLE))
+    else
+        echo "Cannot determine CUDA_HOME: cuda-gdb not in PATH"
+        return 1
+    fi
+fi
+
+if [[ ! -d "${CUDA_HOME}" ]]
+then
+    echo "Directory specified in CUDA_HOME(=${CUDA_HOME}) doesn't exist"
+    return 1
+fi
+
+if [[ ! -f "${CUDA_HOME}/lib64/stubs/libcuda.so" ]]
+then
+    echo "File ${CUDA_HOME}/lib64/stubs/libcuda.so doesn't exist"
+    return 1
+fi
+
+if [[ -z "$(${CUDA_HOME}/bin/nvcc --version | grep "Cuda compilation tools, release __PKG_VERSION__")" ]]
+then
+  if [[ "${CONDA_BUILD}" = "1" ]]
```

Comment:
```suggestion
  if [[ "${CONDA_BUILD}" == "1" ]]
```

Double brackets -> double equals :D

Reply:
Thanks, Jaime. In Bash it works fine:
```
$ CONDA_BUILD=1; if [[ "${CONDA_BUILD}" = "1" ]]; then echo "ok"; else echo "not ok"; fi
ok
$ CONDA_BUILD=1; if [[ "${CONDA_BUILD}" == "1" ]]; then echo "ok"; else echo "not ok"; fi
ok
$ CONDA_BUILD=0; if [[ "${CONDA_BUILD}" = "1" ]]; then echo "ok"; else echo "not ok"; fi
not ok
$ CONDA_BUILD=0; if [[ "${CONDA_BUILD}" == "1" ]]; then echo "ok"; else echo "not ok"; fi
not ok
```
But I like double equals better, too.

Reply:
(And by the way, the single equal is the status quo, not my change.)

Reply:
The `=` is a POSIX thing. So would suggest we stick with it ( https://github.com/conda-forge/nvcc-feedstock/pull/91#discussion_r931478978 )